platform/osx/joypad_osx.h

```diff
@@ -68,7 +68,7 @@ struct joypad {
 
 	io_service_t ffservice = 0; /* Interface for force feedback, 0 = no ff */
 	FFCONSTANTFORCE ff_constant_force;
-	FFDeviceObjectReference ff_device;
+	FFDeviceObjectReference ff_device = nullptr;
 	FFEffectObjectReference ff_object;
```

Comment:
I guess this could be initialized to `nullptr` too just to be safe.

Reply:
Done!